test/net/sourceforge/kolmafia/textui/command/HelpCommandTest.java

```diff
@@ -0,0 +1,31 @@
+package net.sourceforge.kolmafia.textui.command;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.containsString;
+
+import net.sourceforge.kolmafia.request.GenericRequest;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+public class HelpCommandTest extends AbstractCommandTestBase {
+  @BeforeEach
+  public void initEach() {
+    // Stop requests from actually running
+    GenericRequest.sessionId = null;
+  }
+
+  public HelpCommandTest() {
+    this.command = "help";
+  }
+
+  @Test
+  void Refresh() {
+    String output = execute("refresh");
+
+    assertContinueState();
+    assertThat(
+        output,
+        containsString(
```

Comment:
Tiny thing but could you change this to just be startsWith `refresh all |`. That would reasonably cover the regression while also not make us have to change this test every time we add a new option to refresh

Reply:
Can you make this `startsWith` though, I'm so sorry!

Reply:
Sure, I overlooked that at first, haha.